internal/service/federatedsettingsidentityprovider/model_federated_settings_identity_provider_test.go

```diff
@@ -375,6 +376,7 @@ func TestFlattenFederatedSettingsIdentityProvider(t *testing.T) {
 					"user_claim":                   nilStringPtr,
 					"description":                  &description,
 					"authorization_type":           nilStringPtr,
+					"idp_type":                     conversion.StringPtr(federatedsettingsidentityprovider.WORKFORCE),
```

Comment:
might be irrelevant, but should we have WORKLOAD somewhere?

Reply:
In the model methods there isn't any kind of validation of the values, so I think it's okay to not test different values for idp_type